e2e/perf/scroll.spec.ts

```diff
@@ -0,0 +1,149 @@
+
+            const interval = 60;
+            const dispatchSimulateWheelEvent = (element) => {
+                const event = new WheelEvent('wheel', {
+                    bubbles: true,
+                    cancelable: true,
+                    deltaY,
+                    deltaX,
+                    clientX: 580,
+                    clientY: 580,
+                });
+                element.dispatchEvent(event);
+            };
+
+            // 持续模拟 wheel 事件
+            const continuousWheelSimulation = (element, interval) => {
+                intervalID = setInterval(function () {
+                    dispatchSimulateWheelEvent(element);
+                }, interval);
+            };
+
+            // 开始持续模拟 wheel 事件
+            continuousWheelSimulation(filteredCanvasElements[0], interval);
+        };
+
+        dispathWheelEvent();
+
+        return new Promise((resolve) => {
+            let frameCount = 0;
+            const startTime = performance.now();
+
+            function countFrames(_timestamp) {
+                frameCount++;
+                if (performance.now() - startTime < duration * 1000) {
+                    requestAnimationFrame(countFrames);
+                } else {
+                    clearInterval(intervalID);
+                    const elapsedTime = (performance.now() - startTime) / 1000;
+                    const fps = frameCount / elapsedTime;
+                    resolve(fps);
+                }
+            }
+
+            requestAnimationFrame(countFrames);
+        });
+    }, { duration, deltaX, deltaY });
+
+    return fps;
+}
+
+const createTest = (title: string, sheetData: IJsonObject, minFpsValue: number) => {
+    test(title, async ({ page }) => {
+        // const browser = await chromium.launch({ headless: false }); // launch browser
+        // const page = await browser.newPage();
+        await page.goto('http://localhost:3000/sheets/'); // local service is ok too, that means the port is 3002.
+        await page.waitForTimeout(2000);
+
+        const windowOfPage = await page.evaluateHandle('window');
+        await test.step('create univer', async () => {
+            await page.evaluate(({ sheetData, window }: any) => {
+                window.E2EControllerAPI.disposeCurrSheetUnit();
+                window.univer.createUniverSheet(sheetData);
+            }, { sheetData, window: windowOfPage });
+
+            // wait for canvas has data
+            await page.waitForFunction(() => {
+                const canvaslist = document.querySelectorAll('canvas');
+                if (canvaslist.length > 2) {
+                    const imgData = canvaslist[2]!.getContext('2d')!.getImageData(40, 40, 1, 1).data;
+                    return imgData[3] !== 0;
+                }
+            });
+        });
+
+        try {
+            const fps = await measureFPS(page, 10, 0, 100);
+            await test.step('fps', async () => {
+                console.log('fps', fps);
```

Comment:
We should track this in a time sequence. Maybe store it to another system? cc @jikkai

Reply:
@jikkai  

The current output format is as shown below; how should I output it to make extraction easier in future?


<img width="314" alt="image" src="https://github.com/user-attachments/assets/5ba8696a-ed98-4618-a756-c6edb18199c5">

Reply:
<img width="398" alt="image" src="https://github.com/user-attachments/assets/56269d90-9120-4413-b8b6-590f09fb76a3">

Reply:
<img width="352" alt="image" src="https://github.com/user-attachments/assets/73a69db9-01de-4de0-be09-d7f364f2e6e2">

Reply:
Add max 10 frame time